src/services/WizardService.ts

```diff
@@ -13,6 +13,7 @@ export interface IWizardStage {
   valid: boolean;
   route: any;
   moduleId: any
+  settings?: {[key: string]: any};
```

Comment:
@IonelLupu this or `Record`?

Reply:
I would go with `Record`. I like to use `Record` for any key-value pair objects